components/director/internal/open_resource_discovery/fixtures_test.go

```diff
@@ -608,6 +641,44 @@ func fixBundleCreateInput() []*model.BundleCreateInput {
 	}
 }
 
+func fixAPIsWithLowerVersion() []*model.APIDefinition {
```

Comment:
Based on the comment below and the removal of the version lowering, these functions can be renamed to `fix<Resource>WithHash()`